### PR TITLE
Add Stylelint support

### PR DESCRIPTION
**Pull request by XhmikosR**
This is a first draft. It requires #986 to be merged/sorted out.

I've disabled many rules, but eventually you should go through them and see what you need and what not @thomaspark. Do note that there are some issues left about duplicate selectors and missing `!default` flags.

TODO: add this in test target so that it runs on CI too.

### CODE REVIEW

File: dist/litera/_bootswatch.scss [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 // Variables ===================================================================
 
+// stylelint-disable-next-line value-keyword-case
 $font-family-serif:      Georgia, Cambria, "Times New Roman", Times, serif !default;
```
Comment (XhmikosR) [inline dist/litera/_bootswatch.scss:8]: @thomaspark is this supposed to be here and not in the theme's _variables.scss?

Comment (thomaspark) [reply dist/litera/_bootswatch.scss:8]: Yup, it should be in _variables. Not sure how it got there...

Comment (XhmikosR) [reply dist/litera/_bootswatch.scss:8]: I'll make a separate PR for this.

**Comment by XhmikosR** [comment]
@thomaspark rebased. Feel free to push to my branch to fix the remaining issues. :)

**Comment by XhmikosR** [comment]
@thomaspark I pushed a few more patches, let me know what you think because the whitespace cleanup one is a subjective one :) I could split the default patch to a separate PR if you prefer too.

Now, only the duplicate selectors ones are left. BTW I'd also move away from BEM for consistency and easier search/replace. We can enforce this if you want later.

**Comment by thomaspark** [comment]
Hey @XhmikosR, the patches look good to me, including the whitespace changes. As far as rules, I'm all for keeping them consistent with Bootstrap.

**Comment by XhmikosR** [comment]
@thomaspark I'd say first thing would be to fix the duplicate selector errors and merge the PR. Then you can adapt with the rest of the rules that I have currently disabled to match the upstream config. Maybe not the order rule, but it's your call :)

**Pull request merged**